packages/eslint-config/api-app.yaml

```diff
@@ -1,6 +1,7 @@
 extends: ./api/eslintrc.base.yaml
 env:
   browser: true
+  jquery: true
```

Comment:
Adding jquery for all apps? :scream: 
Can't you update the `api-app-angular` file instead?

Reply:
Well I thought this file was only use by the API team and we have jquery on both...